content/en/_index.md

```diff
@@ -983,6 +983,12 @@ Examples of portable practices include:
 Code snippets should have reasonable test coverage and all critical code paths
 should have integration tests that test against the production service.
 
+### Dedicated testing per sample {#dedicated-testing-per-sample}
+
+Each test case should cover a single sample.
+
+Multiple related samples may be placed in a shared file with common setup and teardown steps.
```

Comment:
I don't even allow this in .NET as in the testing framework we use in C# you can share setup and teardown with multiple tests files (or classes).

Reply:
Thank you for this perspective. I think we might be able to do the same thing in Golang, too ... worth checking out!